src/db/login.js

```diff
@@ -12,7 +12,7 @@ module.exports = ((db) => {
      */
     async function validCredentials(username, password) {
         const user = getUser(username)
-        if (user === null) {
+        if (typeof user === 'undefined' || user === null) {
```

Comment:
Reconsider where this `undefined` to `null` conversion should happen? One function call deeper, i.e. inside of `getUser`?